portfolio/src/main/webapp/blogs.html

```diff
@@ -31,6 +31,10 @@ <h3 id="blog3">Hidden Talent NO.3</h3>
       <p>I just started learning JavaScript and <em>this</em> portfolio is my first piece of code.<br><br>
           <b>Would you like to write some JavaScript as well? Here is a mini-console:</b></p>
       <button onclick="executeConsoleCode()">SURE!</button>
+      <p><b>Additionally, would you like to search for something on Wikipedia? Please type your keyword below:</b><br></p>
+      <textarea id="search-keyword" name="search-keyword" required placeholder="Your keyword..."></textarea><br>
+      <button onclick="searchWikipedia()">SEARCH</button>
+      <div id="wikipedia-container">...searching...</div>
```

Comment:
If possible, I would maybe just hide this text and/or container until the user presses the search button (since it's not really searching until they enter something)

Reply:
I totally agree! I'll make the change. Thanks!